pkg/test/monitor/server_test.go

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Why not just put this in the pkg/networkservice/common/monitor/ directory using package monitor_test ?

Reply:
Moved to tests package inside monitor